tests/azure_iot/testdata/azure_iot.txt

```diff
@@ -0,0 +1,35 @@
+[!exec:bash] stop
+[!exec:grep] stop
+[!exec:cat] stop
+[!exec:curl] stop
+
+exec docker run  --name azure_iot  dautovri/eden-azure-iot-test
```

Comment:
Why do we need it? To create a local image? 

I'd rather not do "exec docker run" in tests

Reply:
Also we'd like to have the Dockerfile  attached

Reply:
This remind me -- we should NEVER be using private docker images. All images should come from either lfedge or itmo orgs @mydatascience @dautovri

Reply:
@dautovri I pushed you image into itmoeve/eden-azure-iot-test.